packages/client/src/styles/globals.css

```
@@ -7284,12 +7338,18 @@ strong {
 }
 
 :root[data-professor-mari-floating="true"] .mari-conversation-call-mini {
-  bottom: calc(max(var(--mari-safe-area-inset-bottom, env(safe-area-inset-bottom)), 0.75rem) + min(32rem, calc(100vh - 5rem)) + 0.75rem);
+  bottom: calc(
+    max(var(--mari-safe-area-inset-bottom, env(safe-area-inset-bottom)), 0.75rem) + min(32rem, calc(100vh - 5rem)) +
```

Stylelint (17.14.0)

[error] Unexpected newline after "+" (scss/operator-no-newline-after)

(scss/operator-no-newline-after)